plugins/flowdock/db/migrate/20150327144239_add_flowdock_checkbox_to_flowdock_flows.rb

```diff
@@ -0,0 +1,5 @@
+class AddFlowdockCheckboxToFlowdockFlows < ActiveRecord::Migration
+  def change
+    add_column :flowdock_flows, :enabled, :boolean, default: false
```

Comment:
this will disable all existing flows ? -> add with default true and then change to default false